EditorExtensions/Misc/MenuItems/BundleFiles.cs

```diff
@@ -64,8 +64,8 @@ private void BeforeQueryStatus(object sender, params string[] extensions)
 
             _files = ProjectHelpers.GetSelectedFilePaths();
 
-            menuCommand.Enabled = _files.All(file => extensions.Contains(Path.GetExtension(file))) &&
-                                  _files.Count() > 1;
+            menuCommand.Enabled = _files.Any(file => extensions.Contains(Path.GetExtension(file))) &&
+                          _files.Count(file => extensions.Contains(Path.GetExtension(file))) > 1;
```

Comment:
Oh I forgot about that! :)

So this will become: 

```c#
menuCommand.Enabled = _files.Count(file => extensions.Contains(Path.GetExtension(file))) > 1;
```

> Do I need to create a new pull request for this?

No. If you are using command line git, you can do the following:

```powershell
# after making all the changes
cd into\WE2013\directory
git rebase --amend
# close the notepad, or you can change the commit message there save and close
git push -f
# to rewrite the commit history of your fork's branch
# this PR will automatically acquire your fork changes made in this particular branch
```

A pull request (PR) is always attached to your branch, as far as it is opened. So any changes you will make in that branch meanwhile, will be reflected in the PR.